app/controllers/albums.js

```diff
@@ -0,0 +1,17 @@
+const { getAlbums, getAlbum } = require('../services/get_albums');
+const { getPhotos } = require('../services/get_photos');
```

Comment:
It seems to much one service per action (like `create_albums`, `update_albums`), services are external APIs (in most of the cases), makes sense have all this functions in one service. I don't see advantages of split this in differents files.